spec/mailers/devise_mailer_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe NotifyMailer, type: :mailer do
+  
+  describe "user sign up" do
+    let(:user) { FactoryBot.create(:user) }
+    
+    it "should be a valid user" do
+      expect(user).to be_valid
```

Comment:
This is testing the FactoryBot configuration. It should be redundant and not part of this spec